src/Numeric/Units/Dimensional/UnitNames/Internal.hs

```diff
@@ -141,12 +141,12 @@ data Prefix = Prefix
                 -- | The name of a metric prefix.
                 prefixName :: PrefixName,
                 -- | The scale factor denoted by a metric prefix.
-                scaleFactor :: Rational
+                scaleExponent :: Int
```

Comment:
Do you want me to redo this to work around the change to `scaleExponent`? Or leave it as is?

Even in the case of moving to binary prefixes we could conceivably keep this API and add `scaleBase`?